async-profiler.graphqls

```diff
@@ -28,16 +28,27 @@ input AsyncProfilerTaskCreationRequest {
     execArgs: String
 }
 
-# profiler task create result
+# AsyncProfiler task create result
 type AsyncProfilerTaskCreationResult {
-    # if null or empty means the task create success, otherwise get create error reason
-    errorReason: String
+    # if null means the task create success, otherwise get create error reason
+    errorReason: AsyncProfilerTaskCreationErrorType
```

Comment:
We don't need the text message now?

Reply:
> Let's provide an enum to list the error code(extend later). Then, message could be about details.

Sorry, maybe I misunderstood this sentence. I thought we defined the enumeration in graphql first, and then added errCode and errMsg when implementing the code. And I want to know what you want to do

Reply:
error code is used to define which kinds of error, message includes more information, and complete sentences to describe the cases.

Reply:
So what you mean is to add a new errCode field to indicate that "the parameters are incorrect" or "there is already a task being executed"?

Reply:
Did you read what is http error code? It sets up kinds. More details are in the messages.

Reply:
Can I modify it like this? 
```graphql
# AsyncProfiler task create result
type AsyncProfilerTaskCreationResult {
    # if null or empty means the task create success, otherwise get create error reason
    # errorCode: 10000 -> ARGUMENT_ERROR include errorReason(SERVICE_ID_IS_NULL,DURATION_IS_NEGATIVE,EVENT_IS_EMPTY)
    # errorCode: 10001 -> ALREADY_MONITOR include errorReason(ALREADY_PROFILING)
    errorReason: String
    errorCode: Int

    # get task id when create success
    id: String
}
```

Reply:
I think AsyncProfilerTaskCreationResult should be two types, `AsyncProfilerTaskCreated` and `FailToCreateAsyncProfilerTask`.